pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "databooks"
-version = "0.1.0"  # dynamically set in CI
+version = "0.0.0-dev"  # dynamically set in CI
```

Comment:
how come the version of the branch is 0.0.0-dev  and set like that by CI (github actions I guess?)? probably a good reason for it but I’d think it be more like 0.1.?-dev , the version it depends on

Reply:
So I just put `0.0.0-dev` as a placeholder/when you are installing it from the repo like you just did
But in CI (GiHub actions indeed) I will be overwriting the version.py file with `echo "__version__ = <version from git tag>" > databooks/version.py`
And same thing with `poetry version <version from git tag>`.
So pip installed packages will show the correct version that is the same in the tags